packages/x-data-grid/vitest.config.browser.mts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="@vitest/browser/providers/playwright" />
+import { mergeConfig } from 'vitest/config';
+import sharedConfig from '../../vitest.shared.mts';
+import { getTestName } from '../../scripts/getTestName.mts';
+
+export default mergeConfig(sharedConfig, {
+  test: {
+    name: getTestName(import.meta.url),
+    environment: 'browser',
+    browser: {
+      enabled: true,
+      instances: [
+        {
+          browser: 'chromium',
+          launch: {
+            // Required for tests which use scrollbars.
```

Comment:
we take screenshots in those tests? Or is there another reason?

Reply:
There were some tests that were failing because they couldn't find the scrollbar. I guess I changed those in my previous PRs

Reply:
Nope, they are still necessary https://app.circleci.com/pipelines/github/mui/mui-x/90923/workflows/031f6686-943e-4184-8dd0-938ad33c993c/jobs/528973